src/sentry/tagstore/v2/models/eventtag.py

```diff
@@ -0,0 +1,37 @@
+"""
+sentry.tagstore.v2.models.eventtag
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2010-2017 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+from __future__ import absolute_import
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import (Model, BoundedPositiveIntegerField, sane_repr)
+
+
+class EventTag(Model):
+    __core__ = False
+
+    project_id = BoundedPositiveIntegerField()
+    environment_id = BoundedPositiveIntegerField()
+    group_id = BoundedPositiveIntegerField()
+    event_id = BoundedPositiveIntegerField()
+    key_id = BoundedPositiveIntegerField()
+    value_id = BoundedPositiveIntegerField()
+    date_added = models.DateTimeField(default=timezone.now, db_index=True)
+
+    class Meta:
+        app_label = 'sentry'
+        db_table = 'sentry_eventtag_v2'
+        unique_together = (('event_id', 'key_id', 'value_id'), )
+        index_together = (
+            ('project_id', 'key_id', 'value_id'),
+            ('group_id', 'key_id', 'value_id'),
+            ('environment_id', 'key_id', 'value_id'),
```

Comment:
I forget my Postgres. Should we just index all of them together? Will Postgres use a subset of an index if the index is like `project, group, event, key, value` and you query `project, key, value`?

Reply:
It won't use it partially like that. It'd scan down `project_id`, and filter the rest in that case since it'd only use the first item in the tuple. But it can't mix and match items.

Reply:
OK, in that case I guess we need each index then.